test/docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM couchbase:latest
```

Comment:
I think this is ok. couchbase:latest is the latest stable DB. couchbase-server:latest is the dev branch. The couchbase:latest is a db with version 6.0 something which should be compatible with any client. The client we are using is the latest 2.7.x . The next Java client is 3.0 beta so they don't really need to match.

Reply:
You asked why not using fhitchen/clj-cb-testdb directly. That image is build from couchbase:latest along with the configure.sh to customize the admin user, earthen user and ram size to initialize the cluster. This Dockerfile is used to create that clj-cb-testdb.

$ docker build --rm -t clj-cb-testdb .

Reply:
> I think this is ok. couchbase:latest is the latest stable DB. couchbase-server:latest is the dev branch. The couchbase:latest is a db with version 6.0 something which should be compatible with any client. The client we are using is the latest 2.7.x . The next Java client is 3.0 beta so they won't ever match.

Sorry, didn't explained properly the issue :) . I'm thinking if we don't touch this library again in the next few years and someone tries to run this image, more certainly it will not work since will download the latest image but the client will still be the old one. If we set the version to a version we know the client is compatible (actually version 6.0) we will not have this problem.

Reply:
agreed, it's better to lock the cb version to avoid confusion and breaking in the future.

Reply:
Ah, I'm with you now. I have changed it 6.0.3 which is the latest now. Next version is 6.5.0-Beta, I tried it and the tests all passed.